src/notifications/generate.ts

```diff
@@ -96,7 +96,7 @@ export const notificationTitleMap: Record<
     `The collection "<b>${ctx.post.title}</b>" just got updated with new details`,
   dev_card_unlocked: () => 'DevCard unlocked!',
   post_bookmark_reminder: (ctx: NotificationPostContext) =>
-    `Reading reminder! <b>${ctx.post.title}</b>`,
+    `Reading reminder! <b>${ctx.post.title ? ctx.post.title : ctx.sharedPost?.title ? ctx.sharedPost?.title : undefined}</b>`,
```

Comment:
Can we do this with if-else, double ternary makes my head spin 😆

Reply:
Yeah, I agree, maybe a generic function to fetch title that tries to fallback on possible ones.